crates/nu-command/src/filesystem/cd.rs

```diff
@@ -1,5 +1,6 @@
 use crate::filesystem::cd_query::query;
 use crate::{get_current_shell, get_shells};
+use libc::gid_t;
```

Comment:
It failed on CI, adding a line `#[cfg(unix)]` would resolve it

Reply:
I just changed this. Let's see if it works @WindSoilder